spec/models/classification_spec.rb

```diff
@@ -539,6 +539,29 @@
     end
   end
 
+  describe "name2tag" do
+    let(:root_ns)   { "/managed" }
+    let(:parent_ns) { "/managed/test_category" }
+    let(:entry_ns)  { "/managed/test_category/test_entry" }
+    let(:parent) { FactoryBot.create(:classification, :name => "test_category") }
+
+    it "creates parent tag" do
+      expect(Classification.name2tag("test_category")).to eq(parent_ns)
+    end
+
+    it "creates tag with name and ns" do
+      expect(Classification.name2tag("test_entry", 0, parent_ns)).to eq(entry_ns)
```

Comment:
Ouch.  Such call is an abuse of the concept of "namespace", as well of the parent = 0 case (which I'd expect only be intended for /ns + cat, not /ns/cat + entry).

I must say I'd happily see direct `File.join` calls instead of `name2tag`.

Reply:
@cben yea? I was thinking of changing all `File.join` calls to `name2tag` - to make them more consistent

Reply:
thanks again for this change